docs/reference.md

```
@@ -0,0 +1,223 @@
+```
+
+### Individual Tool Access
+
+```nix
+go.tools.govulncheck.latest      # latest compatible
+go.tools.govulncheck."1.1.3"     # pinned version
+```
+
+Incompatible versions evaluate to a `throw` with a clear error message:
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)